README.md

```diff
@@ -89,6 +89,90 @@ You can specify the project files in `polymer.json` so that commands like `polym
 }
 ```
 
+## Dependency Variants
+
+A number of commands use a concept called "dependency variants", which are different sets of dependencies that the project should work with.
+
+Typically, when a project can work with a wide range of major versions of
+dependencies, tests are only run against the most recent versions of the
+dependencies. This leaves compatibility with older versions untested. With
+dependency variants you can specify a a number of narrower sets of versions
+to test against to ensure compatibility.
+
+Variants are specified in a `"variants"` property of `bower.json`. `"variants"`
+is a map of variant name to a patch that's applied to the rest of `bower.json`.
+Each property in a variant overwrites a property in `bower.json` if the property
+is a simple value or an Array, otherwise if the property is an Object, it's
+merged.
+
+The result of patching a variant into the `bower.json` is used to perform a
+separate Bower install into a directory named `bower_components-{variant_name}`.
+
+Example:
+
+```json
+{
+  "name": "example",
+  "dependencies": {
+    "polymer": "Polymer/polymer#1.7 - 2.x",
+    "foobar": "Foo/bar#^2.0.0"
+  },
+  "variants": {
+    "polymer-1": {
+      "dependencies": {
+        "polymer": "Polymer/polymer#^1.7.0"
+      }
+    },
+    "polymer-2": {
+      "dependencies": {
+        "polymer": "Polymer/polymer#^2.0.0"
+      }
+    }
+  }
+}
+```
+
+When using `polymer install`, this results in two additional folders,
```

Comment:
Probably best to mention that you have to pass `--variants` in to `polymer install` for this behavior right off the bat. e.g. 

```
When using `polymer install --variants` ...
```